src/dhcpcd-gtk/wpa.c

```diff
@@ -117,6 +118,14 @@ wpa_configure(DHCPCD_WPA *wpa, DHCPCD_WI_SCAN *scan)
 	memcpy(&s, scan, sizeof(s));
 	s.next = NULL;
 
+	i = dhcpcd_wpa_if(wpa);
+	if (!i)
```

Comment:
`i` is not a boolean, so an explicit NULL check makes the intent clear.
`if (i == NULL)`